### PR TITLE
[CLOSE PR] Fix typos and add Event examples 

### PR DESCRIPTION
**Pull request by Velite12**
### What Does This PR Do?

Fix typos and add Custom Event examples

### CODE REVIEW

**Comment by drxeno02** [comment]
@Velite12 Be sure to add everyone for PR review, otherwise they will not be notified and also won't be able to review your PR. I have done this for you this time. Just remember for next time, thanks!

File: README.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 ## Table of Contents
 
 * [What is MIKROS?](#what-is-mikros)
-* [How Does MIKROS Work?](#how-dose-mikros-work)
+* [How Does MIKROS Work?](#how-does-mikros-work)
```
Comment (drxeno02) [inline README.md:4]: 👍

File: README.md [hunk 2]
```diff
@@ -15,11 +21,11 @@
 ## What Is MIKROS?
 MIKROS is a service aimed at game developers which provides useful information and clear analytics in a sharing ecosystem, also known as data pooling. These insights about user behaviour can help developers understand their clientele better including their spending habits. MIKROS uniquely displays its analytics in a simple format with the goal of connecting game developers and advertisers to a full understanding of the problems they are looking to solve. This is due to all the data being in the MIKROS ecosystem.
 
-<a name="how-dose-mirkos-work"></a>
+<a name="how-does-mikros-work"></a>
```
Comment (drxeno02) [inline README.md:24]: 💯

File: README.md [hunk 3]
```diff
@@ -41,22 +47,496 @@ Now, the client has to include the Mikros SDK into their gaming product and setu
 6. *(Optional)* Mikros collects and sends metadata for better user insights and is also crucial for Analytics purpose. This can be enabled or disabled by the ***Auto Track User Metadata*** option from the Mikros Settings. By default, it is kept enabled and is recommended to keep it that way.
 
 <a name="sample-app-&-documentation"></a>
-## Sample App & Dcumentation
+## Sample App & Documentation
```
Comment (drxeno02) [inline README.md:50]: 👍

**Comment by Velite12** [comment]
@drxeno02 I think editing the title without reloading the page must have changed the reviewers, it also removed the assignees.

File: README.md [hunk 4]
```diff
@@ -41,22 +47,496 @@ Now, the client has to include the Mikros SDK into their gaming product and setu
...
- After clicking the Authentication button you will be presented with 3 options. (Sign-in, Sign-up, and Sign-out)
- 
- 
- After clicking the Sign-up button you will  be presented with this screen and will need a Username, Email, and Password to sign up.
- 
- 
- After clicking the Sign-out button you will be presented with this message. Letting you know you have signed out of the Mikros Sample App.
+This is the Mikros Sample App main screen. The Authentication button will allow you to be authenticated and allow you to use the Mikros Sample App.
+
+![MIKROS MAIN SCREEN, add from Git URL dropdown](Documentation/Mikros-main-screen.png)
```
Comment (drxeno02) [inline README.md:57]: @Velite12 always user lowercasing for images. Rename the image Mikros-main-screen.png -> `mikros_main_screen.png` and update references. Do the same for all images e.g. Mikros-sign-in.png -> `mikros_signin.png` | Mikros-sign-up.png -> `mikros_signup.png` | Mikros-sign-out.png -> `mikros_signout.png` 

Note: In code we treat signin, signup, signout as one word. In business language we use two. It is a matter of convenience.

**Review by drxeno02 — CHANGES_REQUESTED** [review]
This is a decent start. I have made some suggestions. There are more things that need to be updated, but I can do it myself. This gets us 90% there, which is good.

**Review by drxeno02 — APPROVED** [review]

**Comment by Debadri-Redapple** [comment]
Closing due to outdated changes.

**Pull request closed without merging**